TestAssets/TestProjects/TestAppSimple/project.json

```diff
@@ -0,0 +1,12 @@
+{
+  "version": "1.0.0-*",
+  "compilationOptions": {
+    "emitEntryPoint": true
+  },
+  "dependencies": {
+    "Microsoft.NETCore.App": "1.0.0-rc2-*"
```

Comment:
I would argue that it is still standalone, albeit requiring an inferred runtime.